.pre-commit-config.yaml

```diff
@@ -1,33 +1,33 @@
 default_language_version:
-  node: system
```

Comment:
A reminder to please revert the changes to this file so this PR.

Reply:
My git is not commiting my code if I put the pre-commit-config file back to the default one